packages/plexus/src/zoom/ZoomManager.tsx

```diff
@@ -96,6 +98,24 @@ export default class ZoomManager {
     this.resetZoom();
   }
 
+  public zoomIn = () => {
+    const selection = this.selection;
+    if (!selection) {
+      this.updateCallback(zoomIdentity);
+      return;
+    }
```

Comment:
can you elaborate what this is doing?

Reply:
It is just used for Null safety, I just used the same convention for null safety in the rest of code which acts like a reset button itself, in case selection is null.

Reply:
what is "selection" and why should it impact zoom behavior?

Reply:
We are making use of a d3-zoom package. That requires an object of Type Selection (object is defined in the module itself) to implement the zoom behaviour. Basically, selections let's the module know the Element (Here the map) to which it should apply the transformation to (Like, scaling, translation etc)